packages/itwinui-css/src/utils/mixins.scss

```diff
@@ -142,3 +142,24 @@
     @content;
   }
 }
+
+// Hide borders and hover/focus effects, eg. input
+@mixin iui-invisible-borders {
+  border: none;
+  background: none;
+
+  &:focus,
+  &:focus-within,
+  &:hover {
+    outline: none;
+    border: none;
+  }
+}
+
+// Vertical divider for content, eg. input, combobox
+@mixin iui-vertical-divider {
```

Comment:
1. Move to utils? 
2. Create React component?

Reply:
makes sense

you can also handle forced-colors
```css
@media (forced-colors: active) {
  background-color: CanvasText;
}
```

Reply:
We have a vertical divider in the header between the brand button & breadcrumbs.  Can we reuse?